server/chat-plugins/announcements.ts

```diff
@@ -2,20 +2,16 @@
  * Announcements chat plugin
  * By Spandamn
  */
-export class Announcement {
-	readonly activityId: 'announcement';
-	announcementNumber: number;
-	room: Room;
+export class Announcement extends Rooms.MinorActivity {
+	activityid = 'announcement' as ID;
```

Comment:
Is there a reason it's all lowercase? My personal preference is for `activityID`; I guess this is more consistent with `roomid` and `userid`, though.

Reply:
i'd prefer activityID too, but i did it for consistency yeah